File: my-folder/0009-palindrome-number/solution.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0)
            return false;
        else if(reverseNum(x)==x)
            return true;
        else
            return false;
    }
    public int reverseNum(int num){
        int reverse=0;
        while(num!=0){
            int remainder=num%10;
            reverse=reverse*10+remainder;
            num=num/10;
        }
        return reverse;
    }
}

File: my-folder/0009-palindrome-number/solution.py
class Solution:
    def isPalindrome(self, x: int) -> bool:

        if x < 0:
            return False

        reverse = 0
        number  = x

        while number != 0:
            reverse = reverse*10 + number%10
            number  = number // 10

        return reverse == x
        
